src/posts/introduction-webc.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Introduction to WebC
+description: What is WebC and what does it have to do with Eleventy?
+date: 2022-10-15
+meta:
+  authorName: W. Evan Sheehan
+  authorBio: "[Evan](https://darthmall.net/) is a web developer living in Lafayette, CO and working at the [National Oceanic and Atmospheric Administration](https://noaa.gov/), where he built the website for [Science On a Sphere](https://sos.noaa.gov) (using Eleventy) and now builds applications for scientists. You may also sometimes find him playing traditional Irish music on fiddle, playing board games, or out biking."
+---
+
+[WebC](https://github.com/11ty/webc) is a new tool by [Zach Leatherman](https://www.zachleat.com/) (creator of Eleventy) for serializing custom elements at build time. It aggregates component-level CSS and JavaScript, allowing developers to keep their styles and scripts together with the markup as single file components the way you may be used to if you work with JavaScript frameworks such as [Svelte](https://svelte.dev/) or [Vue](https://vuejs.org/).
+
+All without having to ship a single line of JavaScript to the client — unless you want it to.
+
+## WebC in a nutshell
+
+First and foremost: WebC components are just HTML. If you already know how to write HTML, CSS, and JavaScript, you are ready to start working with WebC.
+
+Let’s define our first component: a footer for our site. We’ll create a file called `site-footer.webc` and put in it the following HTML:
+
+```html
+<footer>
+  <p>&copy; 2022 Yours Truly</p>
+</footer>
+```
+
+Now we can use a `<site-footer></site-footer>` custom element and WebC will replace that element with the markup from `site-footer.webc`.
+
+> For the purposes of this article, we won’t concern ourselves with [importing components in WebC](https://github.com/11ty/webc#html-imports-kidding-kinda). When you set up the WebC plugin for Eleventy, you can [configure a directory for global components](https://www.11ty.dev/docs/languages/webc/#global-no-import-components) so that they’re always available.
+
+So if we have a file — `page.webc` — with all of our HTML boilerplate, we insert our footer into that file using this custom element.
+
+```html
```

Comment:
Let's add comments in the code sample to emphasize the template since we can assume `.webc` is an unfamiliar extension and it otherwise looks like an HTML file. (and we know folks skim and may not see the name in the preceding paragraph 😉 )

```
<!-- page.webc -->
```